packages/client/src/components/IntensityForm.tsx

```diff
@@ -22,7 +22,7 @@ const FormWrapper = styled("div")({
   right: 0,
   height: 170,
   width: 200,
-  margin: "auto",
+  margin: "43vh auto",

```

Comment:
`margin: auto` is working in the `Form` component. I am not sure why it is not working here. 
Though the above fix works. I ask (politely) if you could find the reason for that.
Otherwise. it is also an acceptable fix to the issue too. 

https://github.com/87prashant/MindChart/blob/895166b3b4e9f9e87626b551261e67cbe2a5b547/packages/client/src/components/Form.tsx#L19

Reply:
The thing is if you to have margin top or bottom auto you need to specify the height of the parent and if margin left or right is auto then you have to specify the width of the parent.
In intensity form you didn't specify height of its parent which is why it centred horizontally only and not vertically.
Auto keywords calculates according to the height or width specified and if not specified it does nothing.
Hope this helped.

Reply:
Thanks for helping
Here, I got it working by adding `position: "fixed"`, you can reflect this in your PR.
```suggestion
  margin: "auto",
  position: "fixed"
```

Reply:
Okay will do that

Reply:
Yeah, I got it why this also, works because position fixed is setting everything relative to the viewport so it will set margins according to the screen size.